src/compaction_execute_bench.rs

```diff
@@ -291,12 +342,22 @@ fn load_options() -> Options {
         num_ssts: num_ssts.parse::<usize>().expect("invalid num ssts"),
         key_bytes: key_bytes.parse::<usize>().expect("invalid key bytes"),
         val_bytes: val_bytes.parse::<usize>().expect("invalid val bytes"),
+        compaction,
         compression_codec,
     };
     info!("Options: {:?}", options);
     options
 }
 
+fn parse_compaction(as_str: String) -> Compaction {
+    let parts = as_str.split(',');
+    let srs: Vec<_> = parts
+        .map(|p| SourceId::SortedRun(p.parse::<u32>().expect("invalid sr id")))
+        .collect();
+    let dst = srs.last().expect("no srs provided").unwrap_sorted_run();
```

Comment:
Should this pop? Otherwise, won't `srs` include the last `dst` ID in its list?

Reply:
It should include the last SR. We by convention always compact the SRs in sources into the last SR.